pkg/minikube/shell/shell.go

```diff
@@ -155,7 +160,7 @@ func (c EnvConfig) getShell() shellData {
 	return shell
 }
 
-func generateUsageHint(ec EnvConfig, usgPlz, usgCmd string) string {
+func GenerateUsageHint(ec EnvConfig, usgPlz, usgCmd string) string {
```

Comment:
all exported functions need to have comment, is this function used anywhere outside the package ? if not make it private